### PR TITLE
Added contribution guidelines 

### PR DESCRIPTION
**Pull request by bobquest33**
Adding contribution guidelines
Closes: #11
Signed-off-by: Priyabrata Dash (bobquest33) bobquest33@gmail.com

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+- Before posting a support issue please help to check the Readme & also the command help if it answers your question
+- If you idea for a new feature, please consider if it fits into the project goals and it enhances the current features and also see if you can send a  [pull request](https://help.github.com/articles/using-pull-requests).  
+
+### Suggested Labels For Issues
+
+-   **Issue Types**
+    -   **Bug**  - Anything that is broken
+    -   **Regression**  - A bug that did not exist in previous versions and isn't a new feature (applied in tandem with Bug)
+    -   **Feature**  - Anything that involves API changes, should generally only be for PRs or bug reports on in-progress features.
```
Comment (raukadah) [inline CONTRIBUTING.md]: Feature/Enhancement

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+cd linkstatus
+virtualenv .venv
+source .venv/bin/activate
+pip install .
+linkstatus --help
+```
+
+# Your First Contribution
+
+- Unsure where to begin contributing? You can start by looking through these beginner and help-wanted issues:
```
Comment (raukadah) [inline CONTRIBUTING.md]: We can include about issues with labelled with https://github.com/pythonpune/linkstatus/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

**Comment by raukadah** [comment]
Thank you @bobquest33, It is really awesome contributors.md file.

**Comment by bobquest33** [comment]
I have included all the fixes please help to check

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+# Setting up the environment
+
+Below is the steps for setting up a development environment
+```
+git clone https://github.com/pythonpune/linkstatus
+cd linkstatus
+virtualenv .venv
+source .venv/bin/activate
+pip install .
```
Comment (digitronik) [inline CONTRIBUTING.md]: @bobquest33 development env should be in editable mode. It will be great if we include `-e`

**Comment by bobquest33** [comment]
All checks have passed, please help to re-review now

**Pull request merged**